frontend/src/metabase/collections/components/PinnedQuestionCard/PinnedQuestionCard.unit.spec.tsx

```diff
@@ -0,0 +1,54 @@
+import fetchMock from "fetch-mock";
+import {
+  createMockCard,
+  createMockCollection,
+  createMockCollectionItem,
+  createMockDataset,
+} from "metabase-types/api/mocks";
+import { ROOT_COLLECTION } from "metabase/entities/collections";
+import {
+  setupCardEndpoints,
+  setupCardQueryEndpoints,
+} from "__support__/server-mocks";
+import { renderWithProviders, screen } from "__support__/ui";
+import { createMockState } from "metabase-types/store/mocks";
+import { createMockEntitiesState } from "__support__/store";
+import PinnedQuestionCard from "./PinnedQuestionCard";
+
+function setup() {
+  const card = createMockCard({
+    id: 1,
+  });
+  const collectionItem = createMockCollectionItem({
+    id: card.id,
+    model: "card",
```

Comment:
This line change is introduced in another PR for `master` https://github.com/metabase/metabase/pull/35372